src/schema/v2/admin/featureFlags.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLFieldConfig,
+  GraphQLID,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { merge, sortBy } from "lodash"
+import { ResolverContext } from "types/graphql"
+import { date } from "../fields/date"
+import { base64 } from "lib/base64"
+
+export const FeatureFlagType = new GraphQLObjectType<any, ResolverContext>({
+  name: "FeatureFlag",
+  fields: {
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      resolve: (data) => {
+        return base64(data.name)
+      },
+    },
```

Comment:
Relay [GlobalID cache support](https://relay.dev/graphql/objectidentification.htm)

Reply:
Maybe augment the object with `id` in the resolver and then just use the global ID field? https://github.com/artsy/metaphysics/blob/ca3ca865c173d956c551a60db9d70d371c75cd9e/src/schema/v2/object_identification.ts#L166-L178